recipes/ddtrace-python.rb

```diff
@@ -6,6 +6,11 @@
 # NB: This recipe is compatible with Chef <= 12 only. If you're using Chef 13 or higher
 # please refer to the README of the cookbook.
 
+log 'message' do
+  message 'This recipe will be deprecated in version 3.0.0 of the cookbook. See README.md chapter ddtrace-python for more infos.'
```

Comment:
`chapter` -> `section`

`infos` -> `info`

Also, can you remind the name of the cookbook in the message? (`datadog cookbook`). Generally useful.

Reply:
thanks 👍 
I've also repeated the recipe name.